mappings/net/minecraft/world/gen/chunk/ChunkGenerators.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_7511 net/minecraft/world/gen/chunk/ChunkGenerators
+	METHOD method_44219 registerAndGetDefault (Lnet/minecraft/class_2378;)Lcom/mojang/serialization/Codec;
```

Comment:
Should be renamed to `initialize` (see fabricord discussion)

Reply:
This can be fixed in a later pull request.

Reply:
> Should be renamed to `initialize` (see fabricord discussion)

Didn't we decide to use `init` over `initialize`?